app/src/main/java/ca/pkay/rcloneexplorer/Settings/LookAndFeelSettingsFragment.java

```diff
@@ -160,4 +168,13 @@ private void onDarkThemeClicked(boolean isChecked) {
 
         listener.onThemeChanged();
     }
+
+    private void onWrapFilenamesClicked(boolean isChecked) {
+        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context);
+        SharedPreferences.Editor editor = sharedPreferences.edit();
+        editor.putBoolean(getString(R.string.pref_key_wrap_filenames), isChecked);
+        editor.apply();
+
+        listener.onThemeChanged();
```

Comment:
Copy/Paste error?

Reply:
I wasn't sure if onThemeChanged would need to be called to refresh the recyclerViewAdapter with the new parameter. If it's not needed i'll pull it out

Reply:
I don't think its necessary. If you look at the interface implementations ([`CTRL + ALT + B`](https://www.jetbrains.com/help/idea/navigating-through-the-source-code.html#go_to_implementation) in AS), it only recreates the SettingsActivity to apply a custom primary or accent color.